zip.go

```diff
@@ -96,6 +96,7 @@ func zipFile(w *zip.Writer, source string) error {
 			if err != nil {
 				return err
 			}
+			name = strings.Replace(name, string(os.PathSeparator), "/", -1)
```

Comment:
Ok, I think I understand now. You want to turn `name`, which is a filesystem path at this point, into a `/`-separated path.

If so, you should use `filepath.ToSlash` function for that. See its documentation [here](https://godoc.org/path/filepath#ToSlash).